en/docs/api-analytics/architecture.md

```diff
@@ -0,0 +1,7 @@
+# Analytics Architecture
+
+The following diagram illustrates the basic architecture of the Analytics solution.
+
+<a href="{{base_path}}/assets/img/analytics/apim-analytics-simplified.jpg"><img src="{{base_path}}/assets/img/analytics/apim-analytics-simplified.jpg" width="70%" alt="APIM Analytics Simplified Design"></a>
+
+As depicted above, the Gateways will publish analytics statistics directly to the Analytics Cloud over the internet. The Analytics Cloud will have regional deployments to reduce publishing latencies and honor data privacy. The Analytics console is deployed as a separate web application that will query the processing to visualize API statistics.
```

Comment:
I don't see much correlation between the diagram and the description. The diagram has multiple organizations, processing pipeline, datastores, component querying, etc. that does not seem explained clearly enough here.